.pre-commit-config.yaml

```diff
@@ -46,4 +46,20 @@ repos:
 -   repo: https://github.com/pre-commit/mirrors-pylint
     rev: v2.3.1
     hooks:
-    -   id: pylint
+    - id: pylint
```

Comment:
This change is unrelated to what's stated in the PR title. Please make atomic changes as it's important for the project maintainability.

Reply:
Submit it separately.

Reply:
So new PR for this specific change, right?

Reply:
yep